docker-compose.cypress.yml

```diff
@@ -17,22 +17,15 @@ services:
       CYPRESS_BACKEND_URL: $CYPRESS_BACKEND_URL
       SPEC_PATH: $SPEC_PATH
       TEST_ENV: $TEST_ENV
+    depends_on:
+      - backend
+      - frontend
+      - wiremock
+      - nginx
     volumes:
-      - ./cypress/fixtures/:/app/cypress/fixtures/
-      - ./cypress/e2e/:/app/cypress/e2e/
-      - ./cypress/node_modules/:/app/cypress/node_modules/
-      - ./cypress/plugins/:/app/cypress/plugins/
       - ./cypress/screenshots/:/app/cypress/screenshots/
-      - ./cypress/stubs/:/app/cypress/stubs/
-      - ./cypress/support/:/app/cypress/support/
       - ./cypress/videos/:/app/cypress/videos/
-      - ./cypress/cypress.config.js:/app/cypress/cypress.config.js
-      - ./cypress/package.json:/app/package.json
-      - ./cypress/yarn.lock:/app/yarn.lock
-      - ./cypress/e2e.sh:/app/e2e.sh
       - ./.git/:/app/.git/
-    expose:
-      - "8088"
     command:
       ["-s", "$SPEC_PATH", "-r", "$TEST_ENV"]
```

Comment:
By removing [this](https://github.com/CDCgov/prime-simplereport/pull/5970/files#diff-c0d228fa2876d46bc85408729375ebccd6bbd4d5280f08a25f873953a145ce2cL31), you lose the ability to pass this in and have it matter. That list of args allowed overwriting default behavior depending on where and what is needed to run.

Reply:
good catch, I will remove the command

Reply:
yeah I removed those settings to simplify things, most of these settings never change and we too many knobs for it, too many areas to set things and confuse people

Reply:
If we feel the need to remove some of it, I understand, but I'd rather keep the flags for `SPEC_PATH,TEST_ENV,RUN_OPEN,BROWSER`. They feel extremely useful to me and I have used them to run against a remote environments, test other browsers, and run a subset of tests. I don't like the idea of having to change the script to modify that behavior.

Reply:
added the knobs back in.